docs/toml.md

```diff
@@ -110,7 +110,23 @@
 #     CertFile = "integration/fixtures/https/snitest.org.cert"
 #     KeyFile = "integration/fixtures/https/snitest.org.key"
 #
-
+# To enable basic auth on an entrypoint
+# with 2 user/pass: test:test and test2:test2
+# Passwords are encoded in MD5: you can use htpasswd to generate those ones
```

Comment:
In fact, you can use MD5, SHA1 and BCrypt: https://github.com/abbot/go-http-auth/blob/master/basic.go#L25. I will update the doc.